### PR TITLE
v2 slippage fix

### PR DESCRIPTION
**Pull request by NoahZinsmeister**

### CODE REVIEW

File: src/components/ExchangePage/index.tsx [hunk 1]
```diff
@@ -219,12 +219,25 @@ function ExchangePage({ sendingInput = false, history, params }: ExchangePagePro
...
+  // for each hop in our trade, take away the "innate" price impact from 0.3% fees
+  // e.g. for 3 tokens/2 hops: 1 - ((1 - .03) * (1-.03))
+  const baseFee = basisPointsToPercent(10000 - 30)
+  const realizedFee = !trade
+    ? undefined
+    : trade.route.path.length === 2
+    ? baseFee
+    : basisPointsToPercent(10000).subtract(
+        new Array(trade.route.path.length - 2)
+          .fill(0)
```
Comment (moodysalem) [inline src/components/ExchangePage/index.tsx:231]: is fill needed?

Comment (NoahZinsmeister) [reply src/components/ExchangePage/index.tsx:231]: yeah, i tested it with an array of empties and it didn't trigger the reduce, sort of strange

Comment (moodysalem) [reply src/components/ExchangePage/index.tsx:231]: guess we could have just used `trade.route.pairs`

Comment (NoahZinsmeister) [reply src/components/ExchangePage/index.tsx:231]: yeah, might have been cleaner

**Pull request merged**